### PR TITLE
Minor docstring & prompt fixes for AgentSkills

### PR DESCRIPTION
**Pull request by li-boxuan**
This is a follow-up of #1941

### CODE REVIEW

File: opendevin/runtime/plugins/agent_skills/agentskills.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def _lint_file(file_path: str) -> Optional[str]:
             ret = 'ERRORS:\n'
             ret += result.stdout.decode().strip()
             return ret.rstrip('\n')
-    # Linting skipped. Either the file is not a Python file or auto-linting is disabled.
+    # Not a python file, skip linting
```
Comment (li-boxuan) [inline opendevin/runtime/plugins/agent_skills/agentskills.py:61]: we already checked auto-linting before entering this function

File: tests/integration/mock/CodeActAgent/test_edits/prompt_001.log [hunk 2]
```diff
@@ -50,8 +50,8 @@ edit_file(start: int, end: int, content: str) -> None:
     Edit a file.
     It replaces lines `start` through `end` (inclusive) with the given text `content` in the open file. Remember, the file must be open before editing.
     Args:
-    start: int: The start line number. Must be greater or equal to 1.
-    end: int: The end line number. Must be greater or equal to 1 AND greater than start AND less than or equal to the number of lines in the file.
```
Comment (li-boxuan) [inline tests/integration/mock/CodeActAgent/test_edits/prompt_001.log:54]: > greater than start

Reason for this fix: this is wrong. `end` could be equal to `start`.

Comment (li-boxuan) [reply tests/integration/mock/CodeActAgent/test_edits/prompt_001.log:54]: Also, it was unclear how to evaluate the precedences of "AND" and "or" here. LLM seems to be doing well but I am hoping a more formal statement makes it more clear.

Comment (xingyaoww) [reply tests/integration/mock/CodeActAgent/test_edits/prompt_001.log:54]: This looks much cleaner

File: opendevin/runtime/plugins/agent_skills/agentskills.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 - search_dir(search_term, dir_path='./'): Searches for a term in all files in the specified directory.
 - search_file(search_term, file_path=None): Searches for a term in the specified file or the currently open file.
 - find_file(file_name, dir_path='./'): Finds all files with the given name in the specified directory.
-- edit_file(path, start, end, content): Replaces lines in a file with the given content.
```
Comment (xingyaoww) [inline opendevin/runtime/plugins/agent_skills/agentskills.py:15]: Thanks for spotting the typo!

**Review by xingyaoww — APPROVED** [review]
LGTM!

**Pull request merged**